build.rs

```diff
@@ -0,0 +1,50 @@
+#[cfg(feature = "inline_asm")]
+fn main() {}
+
+#[cfg(all(not(feature = "inline_asm"), not(feature = "stable")))]
+fn check_inline_asm() {
+    compile_error!("Neither feature \"stable\" nor \"inline_asm\" was set!");
+}
+
+#[cfg(all(not(feature = "const_fn"), not(feature = "stable")))]
+fn check_const_fn() {
+    compile_error!("Neither feature \"stable\" nor \"const_fn\" was set!");
+}
+
```

Comment:
Why is it required to set one of these features? Couldn't we just default to non-const functions in this case?

Reply:
removed